modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase

```diff
@@ -55,12 +55,51 @@ definition {
 	}
 
 	@description = "COMMERCE-6160. As a buyer, I want to be able to select only the allowed quantities of a product, if set"
-	@ignore = "true"
 	@priority = "3"
 	test AssertOnlyAllowedQuantitiesAreSelectable {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6160 AssertOnlyAllowedQuantitiesAreSelectable pending implementation
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		CommerceNavigator.searchEntry(entryName = "U-Joint");
+
+		CommerceNavigator.gotoEntry(entryName = "U-Joint");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Configuration");
+
+		AssertElementPresent(
+			key_labelName = "Allowed Order Quantities",
+			locator1 = "TextInput#SPECIFIC_LABEL_TEXT_INPUT");
+
+		Type(
+			key_labelName = "Allowed Order Quantities",
+			locator1 = "TextInput#SPECIFIC_LABEL_TEXT_INPUT",
+			value1 = "1 4 5 7 11");
+
+		Button.clickPublish();
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Account Management");
+
+		CommerceAcceleratorsInitializer.createNewAccountViaFront(
+			accountName = "Test",
+			adminEmailAddress = "test@liferay.com");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		Click(
+			key_productName = "U-Joint",
+			locator1 = "CommerceAccelerators#PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD");
+
+		AssertElementPresent(
+			key_dropdownOption = "U-Joint",
+			locator1 = "AppBuilder#PRODUCT_MENU_SITE_DROPDOWN_OPTION",
+			value1 = "1 4 5 7 11");
```

Comment:
The locator points at the Product Card name and this line it's not actually checking that the values on the quantity selector are only the allowed ones.
Please try to find a different way to check product card quantity selector option values.

Reply:
Done.